src/PhoneValidation.tsx

```diff
@@ -0,0 +1,73 @@
+import type { ChangeEvent } from 'react'
+import { useRef, useState } from 'react'
+import BackButton from './components/BackButton'
+import ValidationMessage from './components/ValidationMessage/ValidationMessage'
+// import CorrectValidation from './components/ValidationMessage'
+import './PhoneValidation.scss'
+
+type Props = {}
+
+const PhoneValidation = ({}: Props) => {
+  const inputsRef = [null, null, null, null].map(() => useRef<HTMLInputElement>(null))
+  const [numbers, setNumbers] = useState<number[]>([-1, -1, -1, -1])
+  const buttonRef = useRef<HTMLButtonElement>(null)
+  const [validated, setValidated] = useState<boolean>()
+
+  const OnChangeNumber = (event: ChangeEvent<HTMLInputElement>, index: number) => {
+    if (!event.target.value) return setNumbers((array) => array.map((value, i) => (i === index ? -1 : value)))
+    if (index < 3) inputsRef[index + 1]?.current?.focus()
```

Comment:
```suggestion
    if (index < inputsRef.length - 1) inputsRef[index + 1]?.current?.focus()
```

If possible, I'd try to make everything relative and avoid absolute values